presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestAnalyzer.java

```diff
@@ -1257,6 +1258,36 @@ public void testLambdaWithSubqueryInOrderBy()
     }
 
     @Test
+    public void testLambdaWithInvalidParameterCount()
+    {
+        assertFails(INVALID_PARAMETER_USAGE, "line 1:27: Expected a lambda that takes 1 arguments but got 2", "SELECT apply(5, (x, y) -> 6)");
```

Comment:
"that takes 1 arguments" -- what about "argument(s)" instead of "arguments" ?
(sorry, i missed that earlier)